README.md

```diff
@@ -40,7 +40,7 @@ After every epoch a snapshot of the models weights and biases is saved on disk.
 We are deploying the model by enabling TensorFlow Serving to reduce response time drastically. Check out ![jibjib-query](https://github.com/gojibjib/jibjib-query) to learn more about how we implemented TensorFlow Serving for our model.
 
 ### New: Convolutional LSTM for Sequence Classification
-In train_LSTM.py we provide a Convolutional LSTM for audio event recognition. Similar to vggish_train.py it performs classification tasks on mel spectrograms. In contrast to vggish_train.py, it does not perform a classification for each spectrogram but analyzes array of matrices and then performs a classification on the entire sequence. C-LSTMs may outperform CNNs when data only contains sparse specific features that don't occure in every timestep.
+In train_LSTM.py we provide a Convolutional LSTM for audio event recognition. Similar to vggish_train.py it performs classification tasks on mel spectrograms. In contrast to vggish_train.py, it does not perform a classification for each spectrogram but analyzes array of matrices and then performs a single classification on the entire sequence. C-LSTMs may outperform CNNs when data only contains sparse specific features that don't occure in every timestep.
```

Comment:
Couple of things:

1. Moving forward I would like to introduce a line break after 80 characters (you can set this in VSCode). It makes reviewing individual lines easier and is also more convenient for users with smaller resolutions.
2. If you reference files, please use code formatting:
> In `train_LSTM.py`we provide a ...
3. Typo:
>  but analyzes **an** array
4. Claims such as ` C-LSTMs may outperform CNNs when data only contains sparse specific features that don't occure in every timestep` could use a reference.